.changeset/swift-pots-camp.md

```diff
@@ -0,0 +1,5 @@
+---
+'@mobile-reality/react-native-select-pro': minor
+---
+
+add textInputProps for search input
```

Comment:
add backticks to name of this prop so should be "add `textInputProps` for search input"